extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -92,105 +92,115 @@ const HIDE_MESSAGE_TIMEOUT_MS = 1500;
 /**
  * @param {!Element} element
  * @param {?Element=} children
+ * @param {boolean} isHidden
  * @return {!Element}
  */
-const renderSystemLayerElement = (element, children) => (
-  <aside class="i-amphtml-story-system-layer i-amphtml-story-system-reset">
-    {children}
-    <a class={String(ATTRIBUTION_CLASS)} target="_blank">
-      <div class="i-amphtml-story-attribution-logo-container">
-        <img alt="" class="i-amphtml-story-attribution-logo" />
+const renderSystemLayerElement = (element, children, isHidden) => {
```

Comment:
Note: despite the seemingly large diff, the only changes to this function are the following:
1. There is now an `isHidden` parameter
2. The system layer element is placed into a `systemLayerElement` variable before being returned
3. The 'i-amphtml-story-hidden' class is conditionally added at the very bottom of the function